src/components/nav.js

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { CgProfile } from 'react-icons/cg';
+import './styles/nav.css';
 
 const Nav = () => (
-  <nav>
-    <h1>Bookstore CMS</h1>
-    <ul>
-      <li>
-        <NavLink to="/">
+  <nav className="nav-container">
+    <h1 className="nav-heading">Bookstore CMS</h1>
+    <ul className="nav-ul">
+      <li className="nav-li">
+        <NavLink to="/" className="links">
           Books
         </NavLink>
       </li>
-      <li>
-        <NavLink to="/categories">
+      <li className="nav-li">
+        <NavLink to="/categories" className="links">
           Categories
         </NavLink>
       </li>
     </ul>
+    <div className="profile-wrap">
+      <CgProfile className="profile-icon" />
+    </div>
```

Comment:
- [ ] Please use the same user icon as mentioned in the Zaplin design but your design is not the same as Zaplin's design.

**Your Design**
![Screenshot from 2023-07-20 19-12-09](https://github.com/zuhebahmed88091/Bookstore/assets/45226848/06d7f001-e530-4744-888d-50adba0057be)

**Zaplin Design**
![Screenshot from 2023-07-20 19-12-00](https://github.com/zuhebahmed88091/Bookstore/assets/45226848/37fd2742-6fc3-473d-9c52-a7fff4d3bad7)

Reply:
I have tried to find the exact icon everywhere in:

- FontAwesome
- Bootstrap
- etc.
- Even in your repo

Unfortunately I could not find this exact icon for **free** anywhere. This one is available in Pro version.
So I have tried my best till now and applied the closest that goes similar to the resemblance to the mentioned icon.